atom/browser/osr/osr_web_contents_view.cc

```diff
@@ -100,16 +103,16 @@ OffScreenWebContentsView::CreateViewForWidget(
         render_widget_host->GetView());
   }
 
-  auto* relay = NativeWindowRelay::FromWebContents(web_contents_);
+  auto relay = NativeWindowRelay::FromWebContents(web_contents_);
```

Comment:
another chromium-style issue—please leave the `*`. See [here](https://groups.google.com/a/chromium.org/forum/#!msg/chromium-dev/MVLrzzu3DcM/uQWzpe5BBQAJ;context-place=topic/chromium-dev/PMxDx7eR_YY) for a discussion on why auto reducing to a raw pointer is disallowed in chromium code.

(also below on L115)